addons/base_import/models/base_import.py

```diff
@@ -201,9 +201,9 @@ def _read_xls(self, options):
     def _read_xls_book(self, book):
         sheet = book.sheet_by_index(0)
         # emulate Sheet.get_rows for pre-0.9.4
-        for row in pycompat.imap(sheet.row, range(sheet.nrows)):
+        for row_index, row in enumerate(pycompat.imap(sheet.row, range(sheet.nrows))):
```

Comment:
may I suggest to use `rowx` for the row index, as per [xlrd convention](https://xlrd.readthedocs.io/en/latest/api.html#xlrd-sheet)?